Slack.Automation/Promact.Core.Repository/ScrumRepository/ScrumBotRepository.cs

```diff
@@ -383,7 +377,51 @@ public class ScrumBotRepository : IScrumBotRepository
         }
 
 
-
+        /// <summary>
+        /// Used to add channel manually by command "add channel channelname"
+        /// </summary>
+        /// <param name="ChannelName"></param>
+        /// <param name="ChannelId"></param>
+        /// <param name="Username"></param>
+        /// <returns></returns>
+        private async Task<string> AddChannelManually(string ChannelName, string Username, string ChannelId)
+        {
+            var returnMsg = string.Empty;
+            if (ChannelId.StartsWith(StringConstant.GroupNameStartsWith, StringComparison.Ordinal))
```

Comment:
There is no comment in the code to understand your logic. Please write appropriate comments at every place like this where the explanation is required. Also, You can create a method to check channel type like IsPublicChannel, This will allow re-use of code and better implementation.